src/components/editions/headline.tsx

```diff
@@ -52,8 +52,38 @@ const standardFontStyles = css`
 		${headline.medium({ lineHeight: 'tight' })}
 	}
 `;
+const interviewStyles = css`
+	margin-left: 0.75rem;
+	border: 0;
+`;
+
+const interviewFontStyles = css`
+	${articleWidthStyles}
+
+	${headline.xsmall({ lineHeight: 'regular' })}
+		font-weight: 400;
+
+	${from.mobileMedium} {
+		${headline.small({ lineHeight: 'regular' })}
+		font-weight: 400;
```

Comment:
We probably don't need to re-specify this value here as its set above.

Reply:
I thought that but, because the headline has a default weight of 500, it gets overwritten. Ideally we might be able to get designs to use Source standard weights but thats tbc.

Reply:
Oh sorry - that's annoying! Yeah agreed.